lib/chewy/type/import.rb

```diff
@@ -150,8 +151,10 @@ def empty_objects_or_scope?(objects_or_scope)
 
         def import_linear(objects, routine)
           ActiveSupport::Notifications.instrument 'import_objects.chewy', type: self do |payload|
+            progressbar = ProgressBar.create total: adapter.import_count(objects) if routine.options[:progressbar]
             adapter.import(*objects, routine.options) do |action_objects|
               routine.process(**action_objects)
+              progressbar.progress += action_objects.values.flatten.length if routine.options[:progressbar]
```

Comment:
maybe? `flatten` is bad, maybe `flatten(1)` instead?
```suggestion
              progressbar.progress += action_objects.map { _, v| v.size }.sum if routine.options[:progressbar]
```

If you chose to apply it, do it in other places too.

Reply:
Thank you! I applyed this change. 

Is this approach better because it is more explicit? @barthez